package.json

```diff
@@ -42,6 +42,7 @@
 		"camelcase": "^6.2.0",
 		"core-js": "^3.8.1",
 		"debounce": "^1.2.0",
+		"jquery": "^3.5.1",
```

Comment:
This is a no-go. We deprecated jQuery and don't want to use it. Injecting a simple button doesn't require jQuery at all.

Reply:
Resolved by 46cf5ee4dac9d154eece5e99834ed7f60261d254